templates/content/taxonomy-term--localgov-blog-author.html.twig

```diff
@@ -0,0 +1,47 @@
+{#
+/**
+ * @file
+ * Theme override to display a taxonomy term.
+ *
+ * Available variables:
+ * - url: URL of the current term.
+ * - name: (optional) Name of the current term.
+ * - content: Items for the content of the term (fields and description).
+ *   Use 'content' to print them all, or print a subset such as
+ *   'content.description'. Use the following code to exclude the
+ *   printing of a given child element:
+ *   @code
+ *   {{ content|without('description') }}
+ *   @endcode
+ * - attributes: HTML attributes for the wrapper.
+ * - page: Flag for the full page state.
+ * - term: The taxonomy term entity, including:
+ *   - id: The ID of the taxonomy term.
+ *   - bundle: Machine name of the current vocabulary.
+ * - view_mode: View mode, e.g. 'full', 'teaser', etc.
+ *
+ * @see template_preprocess_taxonomy_term()
+ */
+#}
+{% if not localgov_base_remove_css %}
+  {{ attach_library('localgov_base/term') }}
+  {#{{ attach_library('localgov_base/blog') }}#}
+{% endif %}
+
+<div{{ attributes }}>
+  <div class="lgd-term--content">
```

Comment:
We should only have `lgd-term--content` if we have it at the same level as `lgd-term`, it's a modifier of the block `lgd-term`, so something like `lgd-term lgd-term--content` or if it's an element, `lgd-term__content` (making sure we have `lgd-term` as an ancestor.